src/redux/books/books.js

```diff
@@ -0,0 +1,33 @@
+import { useReducer, useState } from 'react';
+
+const ACTIONS = {
+  ADD_BOOK: 'add-book',
+  REMOVE_BOOK: 'remove-book',
+};
+
+const reducer = (books, actions) => { //eslint-disable-line
+  switch (actions.type) {
+    case ACTIONS.ADD_BOOK:
+      return [...books, {
+        id: Date.now(),
+        Title: actions.payload.bookTitle,
+        Author: actions.payload.bookAuthor,
+      }];
+    case ACTIONS.REMOVE_BOOK:
+      books.filter((book) => book.id !== actions.payload.id);
+      break;
+    default:
+      return books;
+  }
+};
+
+const [books, dispatch] = useReducer(reducer, []); //eslint-disable-line
```

Comment:
- [ ] **Required:** The following errors prevented your page from being rendered. Kindly fix them.

![image](https://user-images.githubusercontent.com/27709832/200526303-d839ea30-5736-4d1e-ac0a-74a23054dbe3.png)